### PR TITLE
Replaced localhost:8000 with app.wallabag.it

### LINKED ISSUE #18: Fix link to api methods in developer/api/methods.md
The link in the bottom here: https://doc.wallabag.org/en/developer/api/methods.html#other-methods
currently points to http://localhost:8000/api/doc

Maybe this link should point to an existing site like wallabag.it api?

### PR DESCRIPTION
**Pull request by nicosomb**
Fix #18 

(I know that there are some duplicates in the french doc, we need to clean up but we'll merge #53 before).

### CODE REVIEW

File: de/developer/api/methods.md [hunk 1]
```diff
@@ -2,7 +2,7 @@ Deinen ersten Eintrag hinzufügen
 --------------------------------
 
 Dokumentation für diese Methode:
-<http://localhost:8000/api/doc#post--api-entries>.{\_format}
+https://app.wallabag.it/api/doc#post--api-entries.{_format}
```
Comment (Strubbl) [inline de/developer/api/methods.md]: the underscores, don't they need to be escaped anymore? I remember the online docs made the word "format" underlined without the escaping

File: en/developer/api/methods.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 Documentation for this method:
 
-`http://localhost:8000/api/doc#get--api-entries.{_format}`
+http://app.wallabag.it/api/doc#get--api-entries.{_format}
```
Comment (Strubbl) [inline en/developer/api/methods.md]: https?

File: en/developer/api/oauth.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Creating a new API client
 
 In your wallabag account, you can create a new API client at this URL
-http://localhost:8000/developer/client/create.
+https://app.wallabag.it/developer/client/create.
```
Comment (Strubbl) [inline en/developer/api/oauth.md]: Just a suggestion: maybe we should provide only the path `/developer/client/create` here?

File: fr/developer/api.md [hunk 4]
```diff
@@ -89,7 +89,7 @@ Récupérer les articles existants
 --------------------------------
 
 Documentation pour cette méthode :
-<http://localhost:8000/api/doc#get--api-entries>.{\_format}
+https://app.wallabag.it/api/doc#delete--api-entries-{entry}.{_format}
```
Comment (Strubbl) [inline fr/developer/api.md]: you changed from get to delete here. was that your intention?

File: de/developer/api/methods.md (hunk 1 above)
Comment (nicosomb) [reply de/developer/api/methods.md]: I don't think.

File: fr/developer/api.md (hunk 4 above)
Comment (nicosomb) [reply fr/developer/api.md]: Oops 💤

**Pull request merged**